components/scoped-notification/__examples__/light.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ScopedNotification from '~/components/scoped-notification';
+
+class Example extends React.Component {
+	render() {
+		return (
+			<div
+				style={{
```

Comment:
Please remove extra styling.

Reply:
I have removed extra styling from both light and dark examples